CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -470,10 +458,9 @@ class CameraFragment : Fragment() {
 
         // In the background, load latest photo taken (if any) for gallery thumbnail
         lifecycleScope.launch(Dispatchers.IO) {
```

Comment:
this can be `lifecycleScope.launch {` now since `getLatestImageFilename` is main thread safe